tests/lib/AccountStoreIndexeddb.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('AccountStore', () => {
+    it('is a singleton', () => {
+        const instance1 = AccountStore.instance;
+        const instance2 = AccountStore.instance;
+        expect(instance1).toBe(instance2);
+    });
+
+    it('connection can be opened and closed', async () => {
```

Comment:
Naming should follow the "it ..." syntax, for example **it** `can open and close a connection`.